docs/deployment/settings.md

```diff
@@ -451,6 +451,7 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 
 |                           Key                            |      Default      |                                                                                                                    Meaning                                                                                                                     |   Type   | Since |
 |----------------------------------------------------------|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|-------|
+| kyuubi.server.admin.users                                                   || Comma-separated list of Kyuubi service admin users. We use this config to grant admin permission to any service accounts.                                                                                                                      | seq      | 1.8.0 |
```

Comment:
how about just `kyuubi.server.administrators`?

A comma-separated list of Kyuubi service administrators.